userguide/tutorials/getting_started.adoc

```diff
@@ -136,48 +136,176 @@ You can also go to http://<IP>:8080/api.html to explore the KillBill APIs.
 
 === Tomcat
 
-Users familiar with Java technologies can also install Kill Bill directly in their Web container of choice, such as Tomcat. We recommend using Tomcat version 8.5.
+Users familiar with Java technologies can also install Kill Bill directly in the Web container of their choice. We recommend using Tomcat version 8.5. This section covers the instructions for installing killbill in Tomcat. If this approach is followed, you will also need to configure the database manually as explained below.
 
-==== Tomcat Installation Steps
+==== Tomcat Installation and configuration
 
-If you don't have it yet, you can download the Tomcat container http://tomcat.apache.org/download-80.cgi[here] (get the Core binary distribution). The documentation is available http://tomcat.apache.org/tomcat-8.0-doc/index.html[here].
+The first step in installing Killbill in Tomcat is to download and install Tomcat. For this, you need to follow the steps given below:
 
-In the rest of this guide, we will assume you have installed Tomcat in a directory called *CATALINA_HOME*.
+. Download the Tomcat container from http://tomcat.apache.org/download-80.cgi[here] (get the Core binary distribution). The downloaded zip file contains a folder called *apache-tomcat-8.5.59*
```

Comment:
Maybe precise that it's 8.5.59 at the time of writing and that it can change in the future as newer versions are released.

Reply:
Fixed